src/main/resources/application.yml

```
@@ -1,73 +1,3 @@
-  default-consumes-media-type: application/json;charset=UTF-8
-
-swagger:
-  url: ${SWAGGER_URL}
-
-jwt:
-  secretKey: ${JWT_SECRET}
-  accessExpiration: ${JWT_ACCESS_EXPIRATION}
-  refreshExpiration: ${JWT_REFRESH_EXPIRATION}
\ No newline at end of file
+  profiles:
+    active: local
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)